app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -28,7 +30,10 @@ def initialize(http_response, config, liveness_checking_enabled = false)
             success: false,
             errors: { network: true },
             exception: e,
-            extra: { backtrace: e.backtrace },
+            extra: {
+              backtrace: e.backtrace,
+              reference: reference,
```

Comment:
This is a catch all, we may not have reference here since here we may not have a response from vendor where `referece` is extracted.